Mathlib/Topology/UniformSpace/Equicontinuity.lean

```diff
@@ -126,6 +147,68 @@ protected abbrev Set.UniformEquicontinuous (H : Set <| β → α) : Prop :=
   UniformEquicontinuous ((↑) : H → β → α)
 #align set.uniform_equicontinuous Set.UniformEquicontinuous
 
+/-- A family `F : ι → β → α` of functions between uniform spaces is
+*uniformly equicontinuous on `S : Set β`* if, for all entourage `U ∈ 𝓤 α`, there is a relative
+entourage `V ∈ 𝓤 β ⊓ 𝓟 (S ×ˢ S)` such that, whenever `x` and `y` are `V`-close, we have that,
+*for all `i : ι`*, `F i x` is `U`-close to `F i x₀`. -/
+def UniformEquicontinuousOn (F : ι → β → α) (S : Set β) : Prop :=
+  ∀ U ∈ 𝓤 α, ∀ᶠ xy : β × β in 𝓤 β ⊓ 𝓟 (S ×ˢ S), ∀ i, (F i xy.1, F i xy.2) ∈ U
+
+/-- We say that a set `H : Set (X → α)` of functions is uniformly equicontinuous on a subset if the
+family `(↑) : ↥H → (X → α)` is uniformly equicontinuous on that subset. -/
+protected abbrev Set.UniformEquicontinuousOn (H : Set <| β → α) (S : Set β) : Prop :=
+  UniformEquicontinuousOn ((↑) : H → β → α) S
+
+lemma EquicontinuousAt.equicontinuousWithinAt {F : ι → X → α} {x₀ : X} (H : EquicontinuousAt F x₀)
+    (S : Set X) : EquicontinuousWithinAt F S x₀ :=
+  fun U hU ↦ (H U hU).filter_mono inf_le_left
+
+lemma EquicontinuousWithinAt.mono {F : ι → X → α} {x₀ : X} {S T : Set X}
+    (hST : S ⊆ T) (H : EquicontinuousWithinAt F T x₀) : EquicontinuousWithinAt F S x₀ :=
```

Comment:
```suggestion
    (H : EquicontinuousWithinAt F T x₀) (hST : S ⊆ T) : EquicontinuousWithinAt F S x₀ :=
```
With dot notation, it's better if the assumption which is used for dot notation comes first, as otherwise this leads to cryptic error messages in some situtations.